master-firmware/src/arms/arms_controller.c

```diff
@@ -17,10 +16,6 @@
 
 
 scara_t left_arm;
```

Comment:
maybe we should rename `left_arm` to like `main_arm` or maybe just `arm`?

Reply:
I guess `arm` is enough.